ROMFS/px4fmu_common/init.d-posix/1033_plane_lidar

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# @name Plane SITL with catapult
+#
+
+sh /etc/init.d-posix/1030_plane
+
+if [ $AUTOCNF = yes ]
+then
+	param set RWTO_TKOFF 0
+
```

Comment:
So where are the specific configurations to use the LIDAR and the distance to bottom?

Reply:
Ah I see you will be working on that (https://github.com/PX4/Firmware/pull/14480#issuecomment-604274564).

Reply:
@TSC21 Done!

Reply:
Where exactly? I still see the same params.

Reply:
I added `param set FW_LND_USETER 1` do you see otherwise?